### DISCUSSION TITLE
Can we use different schemas for different datasets?

### DISCUSSION THREAD

**Discussion by safejace** [sanity-io/sanity#3779] [Requests for comment (RFC)] [answered]
am i reading this right... with v3 can we use different schemas for different datasets? https://beta.sanity.io/docs/platform/studio/config#schema

**Comment by kmelve** [comment, marked as answer]
Yes! You can pass in an array of different configurations to `createConfig`. [Learn more here.](https://beta.sanity.io/docs/platform/studio/v2-to-v3/workspaces)
